Dockerfile

```
@@ -1,5 +1,34 @@
+ARG VERSION=unspecified
+
 FROM python:3.6-stretch
```

We should really pin on patch version. `3.6.14` came out 2021-06-28 so it will already differ from the last time this image was built. There's also a `3.6.14-buster` image if you wanted to jump to the current Debian version at the same time.

```suggestion
FROM python:3.6.14-stretch
```

---

I wasn't able to build this Docker image with `python:3.6-buster`.  I believe this is simply because some Debian packages have changed names, but since it wasn't easy greasy I decided to leave it for another day.